connectors/2.0/_index.md

```diff
@@ -1,31 +1,31 @@
 ---
-title: "Jakarta Connectors 2.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Connectors 2.0"
+date: 2020-10-10
 summary: "Release for Jakarta EE 9"
 ---
 The Jakarta Connectors specification defines a standard architecture for Jakarta EE application components to connect to Enterprise Information Systems.
 
 * [Jakarta Connectors 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jca/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Connectors 2.0 Specification Document](./connectors-spec-2.0-RC2.pdf) (PDF)
-* [Jakarta Connectors 2.0 Specification Document](./connectors-spec-2.0-RC2.html) (HTML)
+* [Jakarta Connectors 2.0 Specification Document](./connectors-spec-2.0.pdf) (PDF)
+* [Jakarta Connectors 2.0 Specification Document](./connectors-spec-2.0.html) (HTML)
 * [Jakarta Connectors 2.0 Javadoc](./apidocs)
-* [Jakarta Connectors 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Connectors 2.0 TCK](https://download.eclipse.org/jakartaee/connectors/2.0/jakarta-connectors-tck-2.0.0.zip)([sig](),[sha](),[pub]())
```

Comment:
The sig, sha, and pub links should be filled in:
https://download.eclipse.org/jakartaee/connectors/2.0/jakarta-connectors-tck-2.0.0.zip.sig
https://download.eclipse.org/jakartaee/connectors/2.0/jakarta-connectors-tck-2.0.0.zip.sha256
https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub

Reply:
OK Done those